monitoring/healthz/src/AudiusSurvivalKit.ts

```diff
@@ -21,14 +21,3 @@ export function buildImageUrls(user: any, cid: string, size: string) {
     .map((u: string) => `${u}/ipfs/${cid}/${size}.jpg`)
   return urls[0]
 }
-
```

Comment:
@stereosteve seems this was last modified 6months ago. so im assuming it's safe to remove